drift/lib/src/sentry_transaction_executor.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:drift/backends.dart';
+import 'package:meta/meta.dart';
+import 'package:sentry/sentry.dart';
+import 'sentry_span_helper.dart';
+
+/// @nodoc
+@internal
+class SentryTransactionExecutor extends TransactionExecutor {
+  final TransactionExecutor _executor;
+
+  final Hub _hub;
+
+  final _spanHelper = SentrySpanHelper(
+    // ignore: invalid_use_of_internal_member
+    SentryTraceOrigins.autoDbDriftTransactionExecutor,
+  );
+
+  final String? _dbName;
+
+  /// @nodoc
+  SentryTransactionExecutor(this._executor, Hub hub, {@internal String? dbName})
+      : _hub = hub,
+        _dbName = dbName {
+    _spanHelper.setHub(_hub);
+    beginTransaction();
+  }
+
+  @override
+  TransactionExecutor beginTransaction() {
+    return _spanHelper.beginTransaction(
+      'transaction',
+      () {
+        return _executor.beginTransaction();
+      },
+      dbName: _dbName,
+    );
+  }
+
+  @override
+  Future<void> rollback() {
+    return _spanHelper.abortTransaction(() async {
+      return await _executor.rollback();
+    });
+  }
+
+  @override
+  Future<void> send() {
+    return _spanHelper.finishTransaction(() async {
+      return await _executor.send();
+    });
+  }
+
+  @override
+  SqlDialect get dialect => _executor.dialect;
+
+  @override
+  Future<bool> ensureOpen(QueryExecutorUser user) {
+    return _executor.ensureOpen(user);
```

Comment:
No wrap here as well as the remaining methods in this class?

Reply:
hm, yes there should be, thx